src/authorization.js

```diff
@@ -415,16 +413,16 @@ class Authorization {
     if (this.virtual) {
       return []
     }
-    var statement
-    var fragment = rdf.namedNode('#' + this.hashFragment())
-    var ns = vocab(rdf)
-    var statements = [
+    let statement
+    let fragment = rdf.namedNode('#' + this.hashFragment())
```

Comment:
this commit based on branch from this PR implements that https://github.com/elf-pavlik/solid-permissions/commit/e439300c217174469dd69067bb0eaeaf354ab88f